rules/st2workroom_test_master_el7.yaml

```diff
@@ -34,3 +34,5 @@
             instance_type: "m3.large"
             distro: "RHEL7"
             pkg_st2: true
+            install_url: "https://raw.githubusercontent.com/StackStorm/st2workroom/master/script/bootstrap-st2"
```

Comment:
inb4, github changes supported protocols and ciphers and this stuff breaks again, right? :P